### PR TITLE
Marginal 1D dashboard

### PR DESCRIPTION
**Pull request by ndmlny-qs**
This commit adds the following functionality.

- A new module that defines a one-dimensional dashboard for inspecting posterior KDEs in an `InferenceData` object.
- A notebook highlighting the dashboard, and how to use it.

### CODE REVIEW

**Comment by aloctavodia** [comment]
The notebook seems to be missing

File: arviz_dashboard/__init__.py [hunk 1]
```diff
@@ -1,4 +1,5 @@
 """Bayesian Dashboards built on top of Panel"""
 __version__ = "0.0.1a"
 
-from .elpd import dashboard_elpd
\ No newline at end of file
+from .elpd import dashboard_elpd
+from arviz_dashboard.marginal.one_d import posterior_marginal1d_dashboard
```
Comment (aloctavodia) [inline arviz_dashboard/__init__.py:5]: Not necessary to change now, but we should think of shorter names for the dashboard functions and if we are going to import arviz_dashboard as `azd` or something similar then we could omit the `dashboard` from the names.

**Review by aloctavodia — APPROVED** [review]
This is great as is.

Something we can have in the future is the possibility to switch between type of plots, for example in preliz we could have kde, histogram, empirical cdf, quantile dot plot (?). Also we could have the option (maybe even default) of showing the a `point_interval` (see for example https://preliz.readthedocs.io/en/latest/examples/param_space_1d_examples.html#distributions).

Also I wonder of ways of making use of the interactivity. How useful will be to use a box/rectangle to select an interval and get the total mass probability inside that interval?

File: arviz_dashboard/__init__.py (hunk 1 above)
Comment (ndmlny-qs) [reply arviz_dashboard/__init__.py:5]: I like this suggestion, and will make the change before mergining.

I'll also take all the suggestions you mentioned above and turn them into tickets for enhancements.

**Pull request merged**